### PR TITLE
Simplified the logic used for importing Python version specific packages

### PR DESCRIPTION
**Pull request by kshitij10496**
Hey @jgonggrijp 
Thank you for this package.

In this PR, I have tried to unify the import statements for Python version specific packages into a single conditional clause.

### CODE REVIEW

File: pip_review/__main__.py [hunk 1]
```diff
@@ -29,12 +24,13 @@ def check_output(*args, **kwargs):
...
+    import urllib2 as urllib_request
+    from subprocess import check_output
     import __builtin__
-    input = getattr(__builtin__, 'raw_input')  # Python2
-except (ImportError, AttributeError):
-    pass
+    input = getattr(__builtin__, 'raw_input')
 
+from pkg_resources import parse_version
 from packaging import version
```
Comment (jgonggrijp) [inline pip_review/__main__.py]: While you're at it, to complete the neatness, would you mind moving these two lines above the if/else block? Thanks in advance.

**Comment by kshitij10496** [comment]
> Thank you for your PR, @kshitij10496 (which is your given name, Kshitij or Saraogi?). It does look a bit neater this way.

I am happy to contribute. BTW, my first name is Kshitij.
I will reorder the import statements as you have suggested and squash the commits, if that works for you.

**Comment by jgonggrijp** [comment]
In this case I don't mind a squash, but in general it is better to leave history unchanged once it has been published. Do what you prefer, I'll merge it either way.

**Comment by kshitij10496** [comment]
> In this case I don't mind a squash, but in general it is better to leave history unchanged once it has been published.

I couldn't agree more.
I have updated the PR. Let me know if any additional changes need to be made.

**Comment by jgonggrijp** [comment]
Oops, I almost missed this: this project uses [git-flow](http://nvie.com/posts/a-successful-git-branching-model/), incidentally by the same author as the original pip-tools. Basically, this means that your branch should be merged into `develop`, not into `master` (as your PR is requesting). This is also mentioned in the `CONTRIBUTING.md`.

I manually merged your branch into `develop`. GitHub will however not recognize this as acceptance until the next release is out, when `master` finally descends from your contribution. So this PR will appear "open" for the time being, even though I have already accepted it.

**Comment by kshitij10496** [comment]
@jgonggrijp That's fine by me.
Thanks for the prompt review and merge.

**Pull request merged**